src/constants.ts

```diff
@@ -26,6 +26,13 @@ export const onlyGitHubSupported: string = localize('onlyGitHubSupported', 'Only
 export const githubAuthProviderId: string = 'github';
 export const githubScopes: string[] = ['repo', 'workflow', 'admin:public_key'];
 
+export const freeSkuDetail: string = localize('freeSkuDetail', 'Free SSL, 2 Custom Domains');
+export const standardSkuDetail: string = localize('standardSkuDetail', 'Free SSL, 5 Custom Domains, Custom Authentication, SLA');
```

Comment:
Looking at the https://azure.microsoft.com/en-us/pricing/details/app-service/static/, I think we should include the $9 per month since that's the only thing that actually describes the pricing.

I'm not sure if we need the Free SSL in the details since that applies to both.

Reply:
> I think we should include the $9 per month

I thought about including the $9/mo pricing, but was worried about the asterisks, since I'm not sure if we are required to display them. And I don't know where to display them. Or are these not an issue?

![image](https://user-images.githubusercontent.com/12476526/128550905-d0bc248f-f063-45ac-995c-f864dca352ab.png)

![image](https://user-images.githubusercontent.com/12476526/128550855-9fcc96b7-8a19-4641-a893-38af83e96ee1.png)

> I'm not sure if we need the Free SSL in the details since that applies to both.

I was using this as an example, and it shows Free SSL on each item.

![image](https://user-images.githubusercontent.com/12476526/128551121-1a2db797-6c30-4061-bed7-1dd9a06a53fe.png)

Reply:
IMO the only thing worse than no pricing is inaccurate pricing. If you can find an API to dynamically get the pricing that's great, but otherwise I don't think we should hard-code a price that is subject to change. Unfortunately we've never been able to find that API for other extensions, but we have added a link to the pricing page at the bottom of the quick pick

Reply:
Azure does have a [Retail prices REST API](https://docs.microsoft.com/en-us/rest/api/cost-management/retail-prices/azure-retail-prices#api-property-details). Seems like we can get pretty specific pricing information, however kinda feels overkill when we could just add link to the pricing page.